adserver/management/commands/archive_offers.py

```diff
@@ -181,13 +183,13 @@ def delete_offers(self, day):
         self.stdout.write(query % (day, end_day))
 
         # Always delete from the default.
-        with connection["default"].cursor() as cursor:
+        with connections["default"].cursor() as cursor:
             # Offers are normally immutable so the delete has to be run as raw sql
             cursor.execute(
                 query,
                 [day, end_day],
             )
-            deleted_offers = cursor.fetchone()
+            deleted_offers = cursor.fetchone() or 0
```

Comment:
I also hit this one :)

Reply:
Shouldn't this be:

> deleted_offers = cursor.rowcount

fetchone() for a DELETE query returns an exception:

> django.db.utils.ProgrammingError: no results to fetch

See discussion in: https://dba.stackexchange.com/questions/75921/delete-returns-nothing-in-psycopg2

Reply:
@mattishaden thanks for catching this. You are correct. I'm not sure how this every worked although we did run it multiple times. Curious...